### ISSUE TITLE
📡(b37): Communication, UI: app does not inform users when server is unreachable or music cannot be loaded

### ISSUE THREAD

**Issue by bradtchapman** [BillGoldsmith/iOS-bugfest#96]
### Description of issue:

Unexpected behavior: The app does not inform the user at all when music fails to load, even if the app and the device otherwise meet the conditions to play music: connected to a valid network, device says the net is reachable, and RP is not blocked by the wifi/cell playback setting.

### Expected behavior:

The app needs to display a message that the RP servers are unavailable at any point during the listening session if this becomes true.  Ideally, before music starts playing, there should be some kind of indicator that it is attempting to load ... something.  If the most recently
downloaded gapless tracks are still playing on device when this becomes true, display a warning icon on screen but allow playback to continue.  

### Workaround: 

None.  This is behavior that has to be addressed within the app.

**Comment by bradtchapman** [comment]
Note; you're going to need to implement a keep-alive or phone-home behavior and use the networking APIs to monitor server availability.  @mtnbrit and @jarro can consult on best practice for checking into the server at scale without overloading it for regular listeners.  

In another ticket, I sent several sessions from past WWDC sessions about designing for unreliable networks.  Please use these as a template.

**Comment by donshefer** [comment]
Error reporting and recovery has been added to build 43.
